tutorial/11 - Running Bokeh Applications.ipynb

```diff
@@ -27,20 +27,339 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The architecture of Bokeh is such that high-level “model objects” (representing things like plots, ranges, axes, glyphs, etc.) are created in Python, and then converted to a JSON format that is consumed by the client library, BokehJS. Using the Bokeh Server, it is possible to keep the “model objects” in python and in the browser in sync with one another, creating powerful capabilities:\n",
+    "The architecture of Bokeh is such that high-level “model objects” (representing things like plots, ranges, axes, glyphs, etc.) are created in Python, and then converted to a JSON format that is consumed by the client library, BokehJS. Using the Bokeh Server, it is possible to keep the “model objects” in python and the browser in sync with one another, creating powerful capabilities:\n",
```

Comment:
I would prefer to keep the parallel sentence structure and keep "in" in both sides of the conjunction to emphasize the location where models are (and that they are distinct)